go/stats/counter_test.go

```diff
@@ -71,3 +71,31 @@ func TestGaugeFunc(t *testing.T) {
 		t.Errorf("want 1, got %v", v.String())
 	}
 }
+
+func TestGaugeFloat64(t *testing.T) {
+	var gotname string
+	var gotv *GaugeFloat64
+	clear()
+	Register(func(name string, v expvar.Var) {
+		gotname = name
+		gotv = v.(*GaugeFloat64)
+	})
+	v := NewGaugeFloat64("f", "help")
+	if gotname != "f" {
+		t.Errorf("want Int, got %s", gotname)
+	}
+	if gotv != v {
+		t.Errorf("want %#v, got %#v", v, gotv)
+	}
+	v.Set(3.14)
+	if v.Get() != 3.14 {
```

Comment:
nit: I personally find stretchr assertions like `assert.Equal(t, 3.14, v.Get())` much easier to read

Reply:
You're absolutely right; I copied+pasted existing tests and kept the original code conventions (I assume this was written way before testify was in use). I'll update.

Reply:
fixed